### PR TITLE
Fix Apache License Header and Intel Copyrights

### PR DESCRIPTION
**Pull request by SamerKhshiboun**
* Tracked on [LRS-653]
* change copyrights and license to meet Apache 2.0 standard (see end of page of: https://www.apache.org/licenses/LICENSE-2.0) 
* applied (automatically with copyright command) all changes above, so many files headers have been changed.
* added (auto-generated) CONTRIBUTING.md to the root directory to meet APACHE license
* fixed pr_check to take license and copyright from LICENSE file
* updated LICENSE file
* updated main.yaml workflow to do first the pr_check before install CMake or any other further steps

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Any contribution that you make to this repository will
```
Comment (Nir-Az) [inline CONTRIBUTING.md]: Let's talk about it.
It's weird that it starts from 5, lets create a fill contributions file (See LibRealSense)
Github can help with a template

File: realsense2_camera/CMakeLists.txt [hunk 2]
```diff
@@ -1,5 +1,16 @@
-# License: Apache 2.0. See LICENSE file in root directory.
-# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+# Copyright 2023 Intel Corporation. All Rights Reserved.
```
Comment (Nir-Az) [inline realsense2_camera/CMakeLists.txt:1]: Where did you take this copyrights template?

Comment (Nir-Az) [reply realsense2_camera/CMakeLists.txt:1]: Oh I saw where from :)

**Comment by Nir-Az** [comment]
Great PR.

Lets just speak about the contributors file

File: CONTRIBUTING.md (hunk 1 above)
Comment (SamerKhshiboun) [reply CONTRIBUTING.md]: Aligned it with librealsense contributing file

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
+> The `ros2-development` branch is considered stable, at all times.
+> If you plan to propose a patch, please commit into the `ros2-development` branch, or its own feature branch.
+
+In addition, please run `pr_check.sh` under `scripts` directory. This scripts verify compliance with project's standards:
+
+1. Every example / source file must refer to [LICENSE](https://github.com/IntelRealSense/realsense-ros/blob/ros2-development/LICENSE)
+2. Every example / source file must include correct copyright notice
+3. For indentation we are using spaces and not tabs
+4. Line-endings must be Unix and not DOS style
+5. Every API header file must be able to compile as the first included header (no implicit dependencies)
```
Comment (Nir-Az) [inline CONTRIBUTING.md]: 5. Not relevant IMO, we do not have an API or a check for that

Comment (Nir-Az) [reply CONTRIBUTING.md]: Beside that look great!

Comment (SamerKhshiboun) [reply CONTRIBUTING.md]: removed

**Review by Nir-Az — APPROVED** [review]
Great!

**Review by Nir-Az — APPROVED** [review]

**Pull request merged**